public/_locales/de/messages.json

```
@@ -594,6 +594,9 @@
     "adblockerIssue": {
         "message": "Irgendwas hält SponsorBlock davon ab, die Videodaten abzurufen. Eine mögliche Ursache sind Werbeblocker. Mehr Infos: https://github.com/ajayyy/SponsorBlock/wiki/Fix-Ad-Blocker-Blocking-SponsorBlock's-Requests"
     },
+    "adblockerIssueUnlistedVideosInfo": {
+        "message": "Wenn Sie dies nicht lösen können, deaktivieren Sie die Einstellung 'Nicht gelistete und private Videos ignorieren', da SponsorBlock nicht in der Lage ist, die Sichtbarkeit für dieses Video abzurufen"
```

There is an issue with the German translation, specifically the pronouns that are being used: While the rest of the German messages use "normal" (as in: non-honorific) pronouns to address the user ("du", "dir", "dich"), this new translation is using the honorific pronoun "Sie". It is bad style to mix honorific and non-honorific pronouns. It's also weird in general (to native German speakers) to have honorifics in user interfaces, except maybe on government websites.

I would suggest this translation instead:

```suggestion
        "message": "Falls du das nicht lösen kannst, dann deaktiviere die Einstellung 'Nicht gelistete und private Videos ignorieren', da SponsorBlock nicht in der Lage ist, die Sichtbarkeit für dieses Video abzurufen"
```